omod/src/main/java/org/openmrs/module/fhir2/web/servlet/FhirRestServlet.java

```diff
@@ -45,6 +50,14 @@
 	protected void initialize() {
 		// ensure properties for this class are properly injected
 		// TODO find a way around this hack!
+		int defaultPageSize = Integer.parseInt(globalPropertyService.getGlobalProperty(FhirConstants.OPENMRS_FHIR_DEFAULT_PAGE_SIZE));
```

Comment:
This should be moved to below the try / catch block or else this is going to throw a NullPointerException